celesta-java/src/test/java/ru/curs/celesta/bad/annotated/callcontextnotfirst/CallContextNotFirst.java

```diff
@@ -0,0 +1,10 @@
+package ru.curs.celesta.bad.annotated.callcontextnotfirst;
+
+import ru.curs.celesta.CallContext;
+import ru.curs.celesta.annotation.CelestaProc;
+
+public class CallContextNotFirst {
+
+    @CelestaProc
+    public void callContextNotFirst(int a, CallContext callContext) {}
```

Comment:
Do we check CallContext injection anywhere in tests? Maybe we should call method somewhere from tests, and call assertNotNull(callContext) inside?

Reply:
This method is incorrect and CelestaProcProvider mus throw CelestaException, cuz callContext is not the first argument. 
I'l add a new tricky test method to check CallContext injection.